waterlog/src/Components/AlertTable.js

```diff
@@ -5,43 +5,50 @@ import { connect } from 'react-redux';
 import { fetchSegmentsLeaks } from '../actions/SegmentLeaksActions';
 
 class AlertTableComponent extends Component {
-	componentWillMount() {
+	componentDidMount() {
 		this.props.fetchSegmentsLeaks();
 	}
-	alertInfo = [ 'DATE', 'DESCRIPTION', 'COST', 'STATUS' ];
 
+	alertInfo = ['DATE', 'DESCRIPTION', 'COST', 'STATUS'];
+
+	getStatusIcon = function (severity) {
+		switch (severity.toLowerCase()) {
+			case 'high':
+				return HighStatusIcon();
+			case 'low':
+				return LowStatusIcon();
+			case 'normal':
+				return MediumStatusIcon();
+			default:
+				return null;
+		}
+	};
 	render() {
-		
+		const { error, loading, leaks } = this.props;
+
+		if (error) {
+			return <div>Error! {error.message}</div>;
+		}
+		if (loading) {
+			return <div>Loading...</div>;
+		}
+
 		return (
 			<table>
 				<thead>
 					<tr>{this.alertInfo.map((info) => <th key={info}>{info}</th>)}</tr>
 				</thead>
 				<tbody>
-					{this.props.leaks.map((alert) => (
+					{leaks.map((alert) => (
 						<tr
-						key={alert.id}
-						className={alert.status === 'Fault' ? 'fault' : ''}
-							onClick={() => (window.location.href =  `alert/segment/${alert.id}`)}
-							>
+							key={alert.id}
```

Comment:
try to leave spaces before and after "=". It increases readability

Reply:
added space before and after '='